src/lua-function-list.json

```diff
@@ -31,21 +31,22 @@
     "clearSpecialExits": "clearSpecialExits(roomID)",
     "clearUserWindow": "clearUserWindow(name)",
     "clearWindow": "clearWindow([windowName])",
+    "closeMapWidget": "closeMapWidget()",
     "closeMudlet": "closeMudlet()",
     "connectExitStub": "connectExitStub(fromID, direction) or connectExitStub(fromID, toID, [direction])",
     "connectToServer": "connectToServer(host, port, [save])",
     "copy": "copy([windowName])",
+    "Create_map_window": "usableId = createRoomID([minimumStartingRoomId])",
```

Comment:
this should be moved to the geyser section on the wiki, it's messing up the scraping as is